### DISCUSSION TITLE
Do you use MCX in MATLAB? Octave? Python or standalone?

### DISCUSSION THREAD

**Discussion by fangq** [fangq/mcx#168] [Polls]
We strive to adapt MCX to run on as many platforms as possible (and take pride in our software portability). We would like to hear from you what is the form of MCX that you use the most.

Please share your feedback!

**Comment by jgroehl** [comment]
We're using the standalone executable but call it from Python, configuring the inputs based on a prior step of our simulation pipeline.
We're also experimenting with different and compound source designs for MCX.
